components/profile-tab-group/profile-tab-group.jsx

```diff
@@ -8,6 +8,7 @@ import HintMessage from '../../components/hint-message/hint-message.jsx';
 import ProjectList from '../../components/project-list/project-list.jsx';
 
 const TABS = {
+  // tab name: [ `project type 1`, `project type 2`, ... ]
```

Comment:
Should this be commented out? Just delete?

Reply:
Ah sorry, bad commenting style lol. That line is meant to be a comment. I can move it to just before `const TABS = {` if that can look less confusing 😂

Reply:
Oh I see now...yeah that looks like a commented out property of TABS.